routes/landingPageRoute.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+
+router.get('/landingPage', (req, res)=> {
+    res.sendFile(path.join(__dirname, '../pageFiles/landingPage.html'));
+  });
+router.post('/auth_landingPage', (req, res) => {
```

Comment:
Why is this a Post request, this API is not needed, use GET /userProfile directly